### ISSUE TITLE
Useful console output

### ISSUE THREAD

**Issue by Jckf** [Jckf/DH-Support#43]
Make it obvious to the server admin that players are successfully connecting their DH modded clients, that LOD generation is working, and whatever else.

**Comment by S0ly** [comment]
yup would be really great, and remove the emojis in the console too please ^^*
